tests/utils/shippable/shippable.sh

```diff
@@ -60,14 +66,42 @@ else
 fi
 
 if [ "${SHIPPABLE_BUILD_ID:-}" ]; then
```

Comment:
This entire block can be removed too, as there is no `SHIPPABLE_BUILD_ID` on AZP. The portion in the else statement can be kept.